kapitan/utils.py

```diff
@@ -212,3 +212,29 @@ def deep_get(x, keys):
                 fd.close()
     for i in output:
         print('{0!s:{l}} {1!s}'.format(*i, l=maxlenght + 2))
+
+def get_directory_hash(directory):
+    '''
+    Compute a sha256 hash for the file contents of a directory
+    '''
+    if not os.path.exists(directory):
+        logger.error("utils.get_directory_hash failed, %s dir doesn't exist", directory)
+        return -1
```

Comment:
Can you raise KapitanError (or similar here https://github.com/deepmind/kapitan/blob/master/kapitan/errors.py) instead?